third_party/meerkat/Component/mmSH/android/java/src/com/samsung/android/meerkat/MeerkatServerService.java

```diff
@@ -167,13 +167,14 @@ private static boolean shouldUseDebugIni() {
 
     public static boolean startCastanetsRenderer(String args) {
         PackageManager packageManager = applicationContext.getPackageManager();
-        Intent intent = packageManager.getLaunchIntentForPackage(
-                applicationContext.getPackageName());
+        Intent intent = new Intent();
+        intent.setClassName("com.samsung.android.castanets","com.google.android.apps.chrome.Main");
```

Comment:
LGTM, offload.js also works with this PR.
How about using applicationContext.getPackageName() instead of raw string?

Reply:
Sure. I'll check and will amend to the commit.

Reply:
Done